pkg/kubelet/kubelet_cadvisor.go

```diff
@@ -23,6 +23,8 @@ import (
 	kubecontainer "k8s.io/kubernetes/pkg/kubelet/container"
 )
 
+// TODO: Move all these methods into the cAdvisor package to make kubelet package a consumer of libraries rather than a source.
```

Comment:
Remove the TODO. Container stats needs a major refactor to support CRI anyway. Don't think a one-off TODO comment will help.